tests/unit/s2n_client_cipher_null_test.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <s2n.h>
+#include "s2n_test.h"
+#include "crypto/s2n_stream_cipher_null.c"
```

Comment:
This test should be called "s2n_stream_cipher_null_test" instead of "s2n_client_cipher_null_test", to match the source file.

Reply:
Whoops, thanks for catching that. Fixed